.kitchen.yml

```diff
@@ -1,65 +1,70 @@
-      provision_command:
+    driver:
+      image: dokken/centos-7
+      pid_one_command: /usr/lib/systemd/systemd
+      intermediate_instructions:
         # stub out /etc/fstab for fb_fstab
-        - touch /etc/fstab
+        - RUN touch /etc/fstab
         # enable EPEL (for stuff like hddtemp)
-        - rpm -Uvh https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
-        # we need dmidecode for the Ohai shard plugin
-        - yum install -y dmidecode
-        # for some reason rsyslog fails to restart cleanly otherwise
-        - yum remove -y rsyslog
-      run_command: /usr/sbin/init
+        - RUN rpm -Uvh https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+        # we need dmidecode for the Ohai shard and virtualization plugins
+        # https://github.com/test-kitchen/dokken-images/pull/32
+        - RUN yum install -y dmidecode
   - name: ubuntu-16.04
-    driver_config:
-      provision_command:
-        # we need dmidecode for the Ohai shard plugin
-        - apt-get install -y dmidecode
-      run_command: /sbin/init
+    driver:
+      image: dokken/ubuntu-16.04
+      pid_one_command: /bin/systemd
+      intermediate_instructions:
+        # we need dmidecode for the Ohai shard and virtualization plugins
+        # https://github.com/test-kitchen/dokken-images/pull/32
+        - RUN apt-get install -y dmidecode
   - name: ubuntu-18.04
-    driver_config:
-      provision_command:
-        # we need dmidecode for the Ohai shard plugin
-        # and apt needs gnugp2, but doesn't depend on it
-        - apt-get install -y dmidecode gnupg2
-      run_command: /sbin/init
+    driver:
+      image: dokken/ubuntu-18.04
+      pid_one_command: /bin/systemd
+      intermediate_instructions:
+        # we need dmidecode for the Ohai shard and virtualization plugins
+        # https://github.com/test-kitchen/dokken-images/pull/32
+        - RUN apt-get install -y dmidecode
   - name: debian-9
-    driver_config:
-      provision_command:
-        # we need dmidecode for the Ohai shard plugin
-        # and apt needs gnugp2, but doesn't depend on it
-        - apt-get install -y dmidecode gnupg2
-      run_command: /bin/systemd
-  - name: debian-sid
-    driver_config:
-      provision_command:
-        # we need dmidecode for the Ohai shard plugin
-        # and apt needs gnugp2, but doesn't depend on it
-        - apt-get install -y dmidecode gnupg2
-      run_command: /bin/systemd
+    driver:
+      image: dokken/debian-9
+      pid_one_commmand: /bin/systemd
+      intermediate_instructions:
+        # we need dmidecode for the Ohai shard and virtualization plugins
+        # https://github.com/test-kitchen/dokken-images/pull/32
+        - RUN apt-get install -y dmidecode
 
 provisioner:
-  name: chef_zero
+  name: dokken
+  chef_log_level: <%= ENV['CHEF_LOG_LEVEL'] || 'auto' %>
+
+transport:
+  name: dokken
 
 suites:
   - name: default
     run_list:
+      - recipe[ohai_backport]
```

Comment:
don't need to run the recipe... the recipe has nothing in it. Just depend on in from fb_init_sample's metadata.

Reply:
I'm aware, but I don't want to add this to `fb_init_sample` because it's strictly a CI fix, it's definitely not something one should run.